resources/categories.json

```diff
@@ -79,6 +79,7 @@
   "Excise Movement Control System testing guide": ["CUSTOMS"],
   "Excise Movement Control System technical interface specification": ["CUSTOMS"],
   "Post and Parcels Service Guide": ["CUSTOMS"],
+  "Trader Goods Profile Service Guide": ["CUSTOMS"]
```

Comment:
There's a comma missing at the end of this line

Reply:
Fixed typo